File: 2019-S2/Proyecto 2/Proyecto2_201700893/Modelos/Simbolo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto2_201700893.Modelos
{
    class Simbolo
    {
        public EnumTipo tipo;
        public Object valor;

        public Simbolo(EnumTipo tipo, Object valor)
        {
            this.tipo = tipo;
            this.valor = valor;
        }

        public enum EnumTipo
        {
            clase, dim1, dim2, dim3 , caracter, cadena, entero, doble, boleano, nulo, error, funcion, metodo
        }
    }
}

File: 2019-S2/Proyecto 1/2019-SegundoSemestre-Compiladores1/src/Modelos/Simbolo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Modelos;

/**
 *
 * @author Eleazar Lopez <Universidad de San Carlos de Guatemala>
 */
public class Simbolo {
    
    public EnumTipo tipo;
    public Object valor;
    
    public Simbolo(EnumTipo tipo, Object valor){
        this.tipo = tipo;
        this.valor = valor;
    }
    
    public enum EnumTipo{
        entero, doble, cadena, caracter, booleano, error, vacio, arreglo, componente
    }
}
